### PR TITLE
Add Recovery Certificate Details Screen (EXPOSUREAPP-7658)

### PR DESCRIPTION
**Pull request by axelherbstreith**
PR that adds the basic Layout for the Recovery Certificate Details Screen

-> Repo Connection will be done in a follow up PR once the repo is ready

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/recovery_certificate_attribute_strings.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" tools:ignore="MissingTranslation">
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values/recovery_certificate_attribute_strings.xml:2]: can we use the strings for Test covid details screen? or should we combine them in one place such as 
`covid_certificate_attribute_strings.xml`?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        val textParams = binding.subtitle.layoutParams as (LinearLayout.LayoutParams)
+        textParams.bottomMargin = (width / 3) + 170
+        binding.subtitle.requestLayout()
+
+        val behavior: AppBarLayout.ScrollingViewBehavior = params.behavior as (AppBarLayout.ScrollingViewBehavior)
+        behavior.overlayTop = (width / 3) + 170
+    }
+
+    private val deleteTestConfirmationDialog by lazy {
+        DialogHelper.DialogInstance(
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt:104]: I think latest trend is to use `MaterialAlertDialogBuilder` directly , can you please use it ?

**Review by mtwalli — COMMENTED** [review]
Looks good 👍

File: Corona-Warn-App/src/main/res/values/recovery_certificate_attribute_strings.xml (hunk 1 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/values/recovery_certificate_attribute_strings.xml:2]: I will check and combine them if possible 👍

**Review by mtwalli — APPROVED** [review]

**Review by BMItr — APPROVED** [review]
👌

**Review by NataliaLemmerth — COMMENTED** [review]
UA approved

**Pull request merged**